.github/workflows/go-test.yaml

```diff
@@ -48,25 +54,41 @@ jobs:
           kubectl -n rook-ceph wait deployment rook-ceph-mon-d --for condition=Available=True --timeout=90s
           kubectl -n rook-ceph wait deployment rook-ceph-mon-e --for condition=Available=True --timeout=90s
 
+      - name: Rbd command
+        run: |
+          set -ex
+          kubectl rook-ceph rbd ls replicapool
 
+      - name: Basic command
```

Comment:
Even this section, could you break it out to one command per section?

Reply:
done